astrophot/models/psf_model.py

```diff
@@ -60,15 +61,11 @@ def __init__(self, *args, **kwargs):
     def initialize(self, target=None, parameters=None, **kwargs):
         super().initialize(target=target, parameters=parameters)
         target_area = target[self.window]
-        if parameters["flux"].value is None:
-            parameters["flux"].set_value(
-                torch.log10(torch.abs(torch.sum(target_area.data)) / target.pixel_area),
-                override_locked=True,
-            )
-        if parameters["flux"].uncertainty is None:
-            parameters["flux"].set_uncertainty(
-                torch.abs(parameters["flux"].value) * 1e-2, override_locked=True
-            )
+        with Param_Unlock(parameters["flux"]), Param_SoftLimits(parameters["flux"]):
+            if parameters["flux"].value is None:
+                parameters["flux"].value = torch.log10(torch.abs(torch.sum(target_area.data)) / target.pixel_area)
+            if parameters["flux"].uncertainty is None:
+                parameters["flux"].uncertainty = torch.abs(parameters["flux"].value) * 1e-2
```

Comment:
I appreciate that this line isn't really changed in this PR.  But I would encourage a short note saying something like you assume a 1% uncertainty floor if uncertainty is not specified, and then defining a named variable to hold `1e-2`.  Bare numbers without documentation make me worried; assigning them to a named variable can be a nice way of documenting what they are.

Reply:
That makes sense to me, I think I did this just as a stop gap in case initial uncertainties aren't provided. I can make it more explicit what the meaning of the `1e-2` is.